test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
My Hope for the TestProject class would that it would represent a model of the current State of the TestProject class, while it's the responsibility of this outer thing to set up that TestProject class according to the scenario. I'm fine with renaming the outer thing if that makes more sense, but moving the methods into TestProject feels like a mix of concerns. 

What do you think?

---
In reply to: [66348239](https://github.com/dotnet/core-setup/pull/97#discussion_r66348239) [](ancestors = 66348239,66107984,66102620)